systemtest/src/test/java/io/strimzi/systemtest/LogSettingST.java

```diff
@@ -47,6 +52,8 @@
     private static final String FATAL = "FATAL";
     private static final String OFF = "OFF";
 
+    private static final String CG_LOGGING_NAME = "cg-logging";
```

Comment:
Is this typo or does CG have another meaning?

Reply:
typo